specification/cognitiveservices/OpenAI.Inference/models/chat.completions.tsp

```diff
@@ -176,75 +201,113 @@ model ChatCompletionsOptions {
   `model`?: string;
 }
 
-@doc("""
-The representation of a single prompt completion as part of an overall chat completions request.
-Generally, `n` choices are generated per provided prompt with a default value of 1.
-Token limits and other settings may limit the number of choices generated.
-""")
-model ChatChoice {
-  @doc("The chat message for a given chat completions prompt.")
-  @projectedName("json", "message")
-  message?: ChatMessage;
-
+@doc("Model representing the common base between chat choice types.")
```

Comment:
May be worth commenting (or adding to doc) that it's not meant to be a user-facing type and should just be consolidated representation for spread (...) operators.